tests/atoms/gaia_apps.js

```diff
@@ -366,48 +374,52 @@ var GaiaApps = {
          .query('AppWindowManager.getActiveWindow').getTopMostWindow();
     }
 
-    let currentEntryPoint = null;
-    // When browser is on a website, app.manifest is null
-    if (app.manifest) {
-      let entryPoints = app.manifest.entry_points;
-      if (entryPoints) {
-        for (let ep in entryPoints) {
-          if (entryPoints[ep].launch_path == app.manifest.launch_path) {
-            currentEntryPoint = ep;
-            break;
-          }
-        }
-      }
-    }
-
+    console.log('app with manifestURL \'' + app.manifestURL + '\' displayed');
     let result = {
       frame: (app.browser) ? app.browser.element : app.frame.firstChild,
       src: (app.browser) ? app.browser.element.src : app.iframe.src,
       name: app.name,
       origin: app.origin,
-      manifestURL: app.manifestURL,
-      entryPoint: currentEntryPoint
+      manifestURL: app.manifestURL
     };
     return result;
   },
 
   /**
-   * Uninstalls the app with the specified name.
-   */
-  uninstallWithName: function(name) {
-    GaiaApps.locateWithName(name, function uninstall(app) {
-      if (typeof(app) === 'object') {
-        let req = navigator.mozApps.mgmt.uninstall(app);
-          req.onsuccess = function() {
-          marionetteScriptFinished(true);
-        };
-        req.onerror = function() {
-          marionetteScriptFinished(req.error);
-        };
-      } else {
-        // App was never installed, so nothing to do here
+  * Install the app with the specified ManifestURL
+  */
+  install: function(manifestURL, packaged) {
```

Comment:
We don't use the `packaged` variable for this PR. Does install_app_packaged work from your side? If so, we can remove that variable and the if>